lib/neo4j/core/cypher_session/adaptors/http.rb

```diff
@@ -16,7 +16,7 @@ def initialize(url, options = {})
           end
 
           def connect
-            @requestor = Requestor.new(@url, USER_AGENT_STRING, self.class.method(:instrument_request))
+            @requestor = Requestor.new(@url, USER_AGENT_STRING, self.class.method(:instrument_request), @options[:faraday_opts] = {})
```

Comment:
Nice catch. That was some post-implementation cleanup I _thought_ was correct, so thanks for spotting